packages/ui/src/components/Stepper/utils.ts

```diff
@@ -36,7 +33,8 @@ export const linkStyleMap = new Map<StepState, Record<string, string>>([
       "data-h2-background-color": `
         base:all:children[.Step__Icon](primary.light)
         base:iap:children[.Step__Icon](primary)
-        base:all:children[.Step__Tail](gray.light)
+        base:children[.Step__Tail](#C5C6C7)
+        base:dark:children[.Step__Tail](#7C89A3)
```

Comment:
Could you add a comment about how these colours are calculated or what they represent?

Reply:
@substrae can you help out here? I actually don't know 😅 I can add the comment if you just provide the information Peter is looking for.

Reply:
Sure thing - Figma allows for multi-layered background colors, so the design leveraged the page background along with an opacity of the respective light and dark colors. Using that opacity in practice in code created transparent semi-overlapping elements which we didn't want, so we color picked the merged colors from the Figma file to create a solid version of what the transparency in the design generated.

Reply:
okay, hopefully my paraphrased explanation is accurate and makes sense for devs 😛 

[document custom hex colour codes](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9628/commits/1c4c91adc0d40030c7734360eefe43b4af58f9d5)

Reply:
Makes sense to me! But I guess I'm the opposite of the target audience 🤣